packages/opencensus-web-core/src/trace/model/tracing.ts

```diff
@@ -23,7 +23,7 @@ export const NOOP_EXPORTER = new NoopExporter();
 /** Main interface for tracing. */
 export class Tracing implements webTypes.Tracing {
   /** Object responsible for managing a trace. */
-  readonly tracer = new Tracer();
+  readonly tracer = Tracer.instance;
```

Comment:
Is this similar to how OpenCensus Node does it?

Reply:
Yes, I saw they do it like this, and it is more understandable that this class is Singleton.

Reply:
Sounds good!